__tests__/app.test.js

```diff
@@ -382,4 +323,45 @@ describe('GET /api/articles (topic query)', () => {
             expect(body.msg).toBe('Bad request: query does not exist')
     })
 })
+})
+
+
+describe("GET /api/articles/:article_id", () => {
+    test("200: when retrieving an article by an id number of 1, return the correct article information as an object", () => {
+        return request(app)
+        .get("/api/articles/1")
+        .expect(200)
+        .then(({ body }) => {
+            const { article } = body;
+            const expectedArticle = {
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 100,
+                article_img_url: "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+                comment_count: 10
+              }
+            expect(article).toEqual(expectedArticle)
+            expect(typeof article.comment_count).toBe('number')
```

Comment:
When testing for a new property that an endpoint now returns, we should create a new test that only tests for that one new property. We shouldn't ever go back and change old tests 👍